admin/sections/fileLoader.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  FileLoaderSchema,
+  fetchFileLoader,
+  updateFileLoader,
+} from "@/client/loader";
+import { Checkbox } from "@/components/ui/checkbox";
+import { SubmitButton } from "@/components/ui/custom/submitButton";
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { PasswordInput } from "@/components/ui/password-input";
+import { toast } from "@/components/ui/use-toast";
+import { cn } from "@/lib/utils";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useEffect } from "react";
+import { useForm } from "react-hook-form";
+import { useQuery } from "react-query";
+
+export const FileLoaderConfig = () => {
+  const loaderForm = useForm({
+    resolver: zodResolver(FileLoaderSchema),
+  });
+
+  const { data: fileLoader, refetch } = useQuery("fileLoader", fetchFileLoader);
+
+  useEffect(() => {
+    if (fileLoader) {
+      loaderForm.reset(fileLoader);
+    }
+  }, [fileLoader]);
+
+  const submitLoaderForm = async (data: any) => {
+    try {
+      await updateFileLoader(data);
+      refetch();
+      toast({
+        className: cn(
+          "top-0 right-0 flex fixed md:max-w-[420px] md:top-4 md:right-4 text-green-500",
+        ),
+        title: "Config updated successfully",
+      });
+    } catch (error) {
+      console.error(error);
+      toast({
+        className: cn(
+          "top-0 right-0 flex fixed md:max-w-[420px] md:top-4 md:right-4 text-red-500",
+        ),
+        title: "Failed to update file loader configuration!",
+      });
```

Comment:
@leehuwuj the toasts are always the same right?  if so how about sending a PR after the release which moves them to a utils class, e.g.:
```suggestion
      toastSuccess({
        title: "Config updated successfully",
      });
    } catch (error) {
      console.error(error);
      toastFail({
        title: "Failed to update file loader configuration!",
      });
```